modules/admin-ui/src/main/webapp/styles/extensions/components/_tables.scss

```diff
@@ -25,6 +25,14 @@
     .main-tbl {
         border-top-left-radius: 0;
         border-top-right-radius: 0;
+
+        .metadata-entry {
```

Comment:
Maybe some more precise name for the style here?

Reply:
It's meant for a single metadata entry in a collection of metadata after all.
How would you prefer it to be named?

Reply:
@staubesv 
> How would you prefer it to be named?